### PR TITLE
refactor(css): teleport-plugin-css now works for both html templates and jsx code chunks

### LINKED ISSUE #332: Plugin for React external stylesheets
There are still considerable numbers of React users that do not use css-modules or css-in-js.

For that, we could have a plugin that extracts the css in a separate file next to the .js file and imports the style file: `import './component.css'`

### PR DESCRIPTION
**Pull request by alexnm**
`react` now supports: `CSS`, `CSSModules`, `InlineStyles`, `StyledJSX`, `StyledComponents`, `JSS`
`preact` now supports: `CSS`, `CSSModules`, `InlineStyles`

also closes #332

### CODE REVIEW

File: packages/teleport-plugin-angular-base-component/src/utils.ts [hunk 1]
```diff
@@ -16,11 +13,7 @@ export const generateComponentDecorator = (uidl: ComponentUIDL, t = types) => {
       t.objectProperty(t.identifier('selector'), t.stringLiteral('app-root')),
       t.objectProperty(
         t.identifier('templateUrl'),
-        t.stringLiteral(`./${getComponentFileName(uidl)}.${FILE_TYPE.HTML}`)
-      ),
-      t.objectProperty(
```
Comment (JayaKrishnaNamburu) [inline packages/teleport-plugin-angular-base-component/src/utils.ts]: Are we going to handle path in project generators alex ?

Comment (alexnm) [reply packages/teleport-plugin-angular-base-component/src/utils.ts]: no, I moved this to the css-plugin

Comment (JayaKrishnaNamburu) [reply packages/teleport-plugin-angular-base-component/src/utils.ts]: But the output file is not referring I think

```ts
import { Component } from '@angular/core'

@Component({
  selector: 'app-root',
  templateUrl: 'contact-form.html',
  styleUrls: ['contact-form.css'],
})
export class ContactForm {
  constructor() {}
}
```

Comment (JayaKrishnaNamburu) [reply packages/teleport-plugin-angular-base-component/src/utils.ts]: And if the css is moved, the template should still follow the same for `tempalteUrl` right --> ``templateUrl: './contact-form.html'``

**Pull request merged**